src/vse_sync_pp/analyzers/analyzer.py

```diff
@@ -201,7 +206,27 @@ def test(self, data):
     def explain(self, data):
         if len(data) == 0:
             return {}
+        dec = data.iloc[0].timestamp
+        # datetime value with resolution of seconds
+        dtv = datetime.fromtimestamp(int(dec), tz=timezone.utc)
+        # Assume that if `dec` is large enough to be 2023 (the year this was
+        # coded), or a later year, then it represents an absolute date-time.
+        # This is >53 years if counting seconds from zero.
+        #
+        # >>> today = datetime.now()
+        # >>> today
+        # datetime.datetime(2023, 9, 8, 16, 49, 54, 735285)
+        # >>> ts = today.timestamp()
+        # >>> ts
+        # 1694188194.735285
+        # >>> ts / (365 * 24 * 60 * 60)
+        # 53.72235523640554
+        if datetime.now().year - dtv.year <= 1:
+            timestamp = dtv.isoformat()
+        else:
+            timestamp = dec
         return {
+            'timestamp': timestamp,
             'duration': data.iloc[-1].timestamp - data.iloc[0].timestamp,
```

Comment:
It is always going to be seconds right? We can just replace duration per duration(s) so that we dont have to deal with the units later.

Reply:
It is always going to be seconds. We don't have to change the key, we lose it anyway on the conversion into JUnit (it becomes the "time" attribute). Not to worry, as the asciidoc generation puts it back for us.